src/components/panel/panel.js

```diff
@@ -544,4 +651,188 @@ MdPanelRef.prototype._createPanel = function() {
  */
 MdPanelRef.prototype._addStyles = function() {
   this._panelContainer.css('z-index', this._config['zIndex']);
+
+  var positionConfig = this._config['position'];
+
+  if (!positionConfig) { return; }
+
+  var isAbsolute = positionConfig.getAbsolute();
+  var panelPosition = isAbsolute ? 'fixed' : 'relative';
```

Comment:
I'll keep an eye out for cases where this won't work. It looks okay here: http://caniuse.com/#feat=css-fixed